### PR TITLE
[CL][Incentives Query][Bug]: Implement support for CL gauges to incentivized pools query

### PR DESCRIPTION
**Pull request by AlpinYukseloglu**
Closes: #5186

## What is the purpose of the change

This PR expands our incentivized pools query to be inclusive of incentivized CL pools.

## Testing and Verifying

- Changes are tested in `x/pool-incentives/keeper/grpc_query_test.go`

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [x] Code comments?
  - [ ] N/A

### CODE REVIEW

File: x/pool-incentives/keeper/grpc_query.go [hunk 1]
```diff
@@ -87,6 +87,23 @@ func (q Querier) IncentivizedPools(ctx context.Context, _ *types.QueryIncentiviz
...
+		if lockableDuration == epochDuration {
+			epochAlreadyLockable = true
+			break
+		}
+	}
+
+	// Ensure that we only add epoch duration if it does not already exist as a lockable duration.
+	if !epochAlreadyLockable {
+		lockableDurations = append(lockableDurations, epochDuration)
+	}
```
Comment (p0mvn) [inline x/pool-incentives/keeper/grpc_query.go:105]: On mainnet, epoch duration is 1 day, and we also have 1-day uptime. How are we going to differentiate between internal per-epoch gauges and 1 day uptime gauges?

**Review by p0mvn — DISMISSED** [review]

File: x/pool-incentives/keeper/grpc_query.go (hunk 1 above)
Comment (mattverse) [inline x/pool-incentives/keeper/grpc_query.go:105]:
```suggestion
	epochDuration := q.incentivesKeeper.GetEpochInfo(sdkCtx).Duration
	for _, lockableDuration := range lockableDurations {
		if lockableDuration == epochDuration {
			lockableDurations = append(lockableDurations, epochDuration)	
			break
		}
	}
```

**Review by mattverse — APPROVED** [review]

File: x/pool-incentives/keeper/grpc_query.go (hunk 1 above)
Comment (p0mvn) [reply x/pool-incentives/keeper/grpc_query.go:105]: FYI, from @jonator 

```
Yeah we need to make sure it only refers to pools and gauges that are internally incentivized via mint incentives

For external incentives, we handle those in the opposite direction: by querying the gauges and linking those to pools

(Which, a separate topic, causes a lot of performance issues)

Like in hindsight, we could have reduced complexity by having the node return all gauges for a pool, internal and external
```

Comment (AlpinYukseloglu) [reply x/pool-incentives/keeper/grpc_query.go:105]: This change seems to add epoch time to lockable durations only if it already exists in there - we want the exact opposite no?

**Review by p0mvn — APPROVED** [review]

**Pull request merged**